examples/examples/publish_subscribe/README.md

```diff
@@ -0,0 +1,32 @@
+# Publish-Subscribe
+
+## Running The Example
+
+Here's an improved description for the publish-subscribe example of Elkodon:
+
+"This example vividly illustrates a robust publisher-subscriber communication
+pattern between two separate processes. The publisher diligently sends two
+messages every second, each containing essential [`TransmissionData`]. On the
+receiving end, the subscriber diligently checks for new data every second.
+
+Whenever new data arrives, the subscriber swiftly acknowledges it by printing
+the received information to the console.
+
+To observe this dynamic communication in action, open two separate terminals
+and execute the following commands:
+
+**Terminal 1**
+
+```sh
+cargo run --example publish_subscribe_subscriber
+```
+
+**Terminal 2**
+
+```sh
+cargo run --example publish_subscribe_publisher
```

Comment:
If feels a bit weird to have on send message at the publisher and and two receive messages at the subscriber. Initially I thought this was a but, only after looking at the code I notice that the publisher indeed publishes two samples in one loop iteration. Shall I create a PR to fix this?